src/app/_components/footer.tsx

```diff
@@ -0,0 +1,60 @@
+                <div className="w-1/5 flex flex-col">
+                    <p className="font-bold">Quick Links</p>
+                    <a href="" target="_blank" className="hover:text-white/80 hover:underline-offset-2 m-1 font-light">AP [subject]</a>
+                    <a href="" target="_blank" className="hover:text-white/80 hover:underline-offset-2 m-1 font-light">AP [subject]</a>
+                    <a href="" target="_blank" className="hover:text-white/80 hover:underline-offset-2 m-1 font-light">AP [subject]</a>
+                    <a href="" target="_blank" className="hover:text-white/80 hover:underline-offset-2 m-1 font-light">AP [subject]</a>
+                    <a href="" target="_blank" className="hover:text-white/80 hover:underline-offset-2 m-1 font-light">AP [subject]</a>
+                    <a href="" target="_blank" className="hover:text-white/80 hover:underline-offset-2 m-1 font-light">AP [subject]</a>
+                </div>
+                <div className="w-1/5 flex flex-col">
+                    <p className="font-bold">Quick Links</p>
+                    <a href="" target="_blank" className="hover:text-white/80 hover:underline-offset-2 m-1 font-light">AP [subject]</a>
+                    <a href="" target="_blank" className="hover:text-white/80 hover:underline-offset-2 m-1 font-light">AP [subject]</a>
+                    <a href="" target="_blank" className="hover:text-white/80 hover:underline-offset-2 m-1 font-light">AP [subject]</a>
+                    <a href="" target="_blank" className="hover:text-white/80 hover:underline-offset-2 m-1 font-light">AP [subject]</a>
+                    <a href="" target="_blank" className="hover:text-white/80 hover:underline-offset-2 m-1 font-light">AP [subject]</a>
+                    <a href="" target="_blank" className="hover:text-white/80 hover:underline-offset-2 m-1 font-light">AP [subject]</a>
+                </div>
+                <div className="w-1/5 flex flex-col">
+                    <p className="font-bold">Quick Links</p>
+                    <a href="" target="_blank" className="hover:text-white/80 hover:underline-offset-2 m-1 font-light">AP [subject]</a>
+                    <a href="" target="_blank" className="hover:text-white/80 hover:underline-offset-2 m-1 font-light">AP [subject]</a>
+                    <a href="" target="_blank" className="hover:text-white/80 hover:underline-offset-2 m-1 font-light">AP [subject]</a>
+                    <a href="" target="_blank" className="hover:text-white/80 hover:underline-offset-2 m-1 font-light">AP [subject]</a>
+                    <a href="" target="_blank" className="hover:text-white/80 hover:underline-offset-2 m-1 font-light">AP [subject]</a>
+                    <a href="" target="_blank" className="hover:text-white/80 hover:underline-offset-2 m-1 font-light">AP [subject]</a>
+                </div>
+                <div className="w-1/5 flex flex-col">
+                    [Logo] AP Students
```

Comment:
This component looks like it should be in its own function. Giant svgs make the overall component look messier than it could be. Move it to it's own AP Students logo.tsx file or something and import.